docs/_data/table/en/common-configuration.yml

```diff
@@ -98,6 +98,10 @@ alluxio.underfs.s3a.secure.http.enabled:
   Whether or not to use HTTPS protocol when communicating with s3. The default value is false.
 alluxio.underfs.s3a.server.side.encryption.enabled:
   Whether or not to encrypt data stored in s3. The default value is false.
+alluxio.underfs.s3a.signer.algorithm:
+  The signature algorithm which should be used to sign requests to the s3 service. This is
+  optional, and if not set, the client will automatically determine it. For interacting with an s3
+  endpoint which only supports v2 signatures, set this to "S3SignerType".
```

Comment:
It looks like it is from here: https://github.com/aws/aws-sdk-java/blob/0b4f751e18962b4bff3587c8257655e5482814c8/aws-java-sdk-s3/src/main/java/com/amazonaws/services/s3/AmazonS3Client.java#L340